ui/src/app/views/card/component.html

```diff
@@ -0,0 +1,58 @@
+<div class="card item-card">
+  <div class="header-area">
+    <clr-dropdown [clrCloseMenuOnItemClick]="false">
+      <button type="button" clrDropdownTrigger>
+        <clr-icon shape="ellipsis-vertical" class="ellipsis-vertical"></clr-icon>
+      </button>
+      <clr-dropdown-menu *clrIfOpen>
```

Comment:
Is this the same dropdown menu for all objects? I did just see the below if statements, but I don't think we have any of the functionality for notebooks yet. 

Also, currently, when you click on a notebook title it opens it. I don't see that here? Are we removing that functionality? If not, we should probably add a `Open in Jupyter` option to the dropdown.

Reply:
My aim for this PR was to get the UI content up without the links to be functional since the way recent work list is to be retrieved is still to be decided and also there is a story in backlog to make notebooks event functional. Having said that i'll be taking a look at opening notebook functionality tomorrow and if its something that can be easily done will add it to this PR.

Reply:
I'm very confused by this class. I thought the scroller content would be defined in a different story. The scope of RW-854 is just the scroller itself that can accept *something* - I think this should go away.

Reply:
Found it. I think `card/component.html` and `card/component.ts` should be removed. That is defined in https://precisionmedicineinitiative.atlassian.net/browse/RW-1009

Reply:
I actually disagree Greg, I think that this PR is considerably more easy to implement with an underlying structure for what format the data should take when inputted. (Neha and I actually talked about this before hand) It is a bit of scope creep for the ticket, but I think it will make our life much easier in the long run.

Reply:
As per my understanding 1009 is to provide the list of elements as  card object as input and feed into Scroll component. By doing this they could see the cards and scroll through them ( as part of this story?)

Reply:
@NehaBroad - that is correct - RW-1009 provides the input to the scroller. This PR defines the input also. That's what I'm concerned about - blurring the lines between what we scope in planning and what we end up implementing. If RW-854 is re-scoped to define the format of the inputs, then I'd be happier. I have a separate set of concerns about that.

Reply:
@blrubenstein If we re-scope RW-854 to cover the input format, I'll look at this with a different perspective. I think the card class needs to be much more generic if that's the case and not constrain the input to the degree that it does.

Reply:
Agreed. That makes sense to me.